datalad/distributed/ora_remote.py

```diff
@@ -1217,7 +1217,9 @@ def _get_obj_location(self, key):
 def main():
     """cmdline entry point"""
     from annexremote import Master
+    from datalad.ui import ui
     master = Master()
     remote = RIARemote(master)
+    ui.set_backend('annex')  # interactive, stdin/stdout will be used for interactions with annex
```

Comment:
Isn't something like this needed as well? https://github.com/datalad/datalad/blob/66fd8be2cadeda3b85665ae63d407e2241e0bb1a/datalad/customremotes/base.py#L119

Not sure, but if so - that's a problem.

Reply:
good catch -- missed that.  I assume you (`AnnexRemote`?) already interfaces progress indication somehow - probably directly for RIA operations without exposing any possible progressbar?  I guess may be a similar subclass/backend needs to be crafted which would either not care about progress bars or provide adapter for `AnnexRemote` interface and not built-in customremotes/base class?.

Reply:
> I assume you (AnnexRemote?) already interfaces progress indication somehow - probably directly for RIA operations without exposing any possible progressbar?

Yes, reporting progress directly to git-annex via `annexremote`.

Reply:
I guess the easiest then is to establish a dedicated UI for it (e.g. "called" `annexremote`) which would somehow swallow or just raise and exception in cat any possible progress bar requested (e.g. if ora invokes some command which it doesn't know leads to establishing some progress bar).
Anyways -- I would probably better get some "authenticated" ORA setup to mimic your case to actually test it all up... or interested @bpoldrack to take over?
also filed https://github.com/datalad/datalad/issues/5873 which might be of help/interest here

Reply:
> I would probably better get some "authenticated" ORA setup to mimic your case to actually test it all up... or interested @bpoldrack to take over?

I hope to come up with HTTPS setup in a docker image for that next week and rewrite/enhance ORA tests. We'll see where that leads ... Once such setup is running, will look into this again.